MANIFEST.in

```diff
@@ -0,0 +1 @@
+include CITATION.cff LICENSE.md README.md
```

Comment:
Looking again, it might be that nowadays it includes python files by default, but possibly nothing else.

Note that we already have a way to include files via setup.py.  Is there a reason for having MANIFEST.in as well?

Reply:
The other question is whether we really want these files directly in the release objects?  They will still be in the repository under the appropriate tag.

Reply:
need to be at user level to get the aggregator version. so they need to come with the release. there's a discussion here in the lab between having the version number and date set by the releaser vs the install setup.py but thats by the by.

I was told to use MANIFEST.in as the defacto way to add files that are not python files into pypi releases. Yet again, I'm not the best to know whom is right, and am following instructions. if there's a better way, please do fill me in.

Reply:
So the issue here is that though you might find these files on PyPI in an archive, you probably won't find them in the installed version because they are outside of the base python package.  If they did get installed, they would overwrite each other as each was installed.  I don't know your "releaser" script so I don't know how this will query for the files, however if the PyPI release knows where the source code repository is, you could get them from there...

Reply:
(admittedly only if the git tag is included also)

Reply:
wont they go into their own tarball? aka fec's readme will be in the fec tarball?
to be honest, if you know of a way to get those files in a poisiton that i can read them from a end user level, id be happy. going to the internet to get data seems really flakey at best

Reply:
Once installed, you don't have access to the tarball that I know of - the tarballs are extracted into the site-packages folder.  If you want to keep the file at the top level of the repository, there isn't much choice but to get them from the repository really.  If you put them in the tarball, you could get the tarball for each release, but that seems like a level once-removed from easy.

Reply:
silly question. if that is the case, cant these files go one directory lower?

Reply:
fixed. removed README.md and added requirements.txt and pypi_to_import as those are needed for the aggregator to work